providers/provider-pojo/src/main/java/io/servicecomb/provider/pojo/schema/PojoProducers.java

```diff
@@ -35,7 +35,7 @@ public void registerPojoProducer(PojoProducerMeta pojoProducer) {
     pojoMgr.register(pojoProducer.getSchemaId(), pojoProducer);
   }
 
-  public Collection<PojoProducerMeta> getProcucers() {
```

Comment:
This is public method , we need to deprecated this method first and add new updated method for it.

Reply:
already added it back